services/ingestion/etl/inforegio/json/src/lib/transform.js

```diff
@@ -103,7 +103,7 @@ export default (record: Object): Project => {
   // Preprocess project locations
   const locationArray = [];
   const countryArray = record.Project_country
-    ? record.Project_country.split('; ')
+    ? record.Project_country.split('; ').filter(country => country)
```

Comment:
Would it make sense to have the split by `;` sign, and chain a method to remove whitespace?

Reply:
We could use `map` and `trim` but the result would probably be the same. I didn't want to alter the logic of the ETL, I just wanted to remove dead values